Library/Formula/ffms2.rb

```diff
@@ -0,0 +1,26 @@
+class Ffms2 < Formula
+  homepage "https://github.com/FFMS/ffms2"
+  url "https://github.com/FFMS/ffms2/archive/2.20.tar.gz"
+  sha1 "bf09d8ed54e5aad86b352b4d335d29da016f2014"
+
+  depends_on "pkg-config" => :build
+  depends_on "ffmpeg"
+
+  # needed for the test to exit cleanly
```

Comment:
Yes it's needed for the `test` block. The patch in the `patch` section is actually my commit which adds this functionality, so no pr or issues are open. It won't be necessary for next version